src/Samples/DIPS.Xamarin.UI.Samples/Controls/Toast/ToastPage.xaml.cs

```diff
@@ -0,0 +1,142 @@
+using System.Threading.Tasks;
+using System.Windows.Input;
+using DIPS.Xamarin.UI.Controls.Toast;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+using Color = System.Drawing.Color;
+using ToastControl = DIPS.Xamarin.UI.Controls.Toast.Toast;
+
+namespace DIPS.Xamarin.UI.Samples.Controls.Toast
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class ToastPage : ContentPage
+    {
+        private string m_pageTitle;
+        private ICommand m_showToastCommand;
+        private ICommand m_showToastCommand2;
+        private ICommand m_showToastCommand3;
+
+        public ToastPage()
+        {
+            InitializeComponent();
+
+            PageTitle = "Hello, World!";
+
+            ShowToastCommand = new Command(parameter =>
+            {
+                ToastControl.Current.ToastLayout = LayoutOneMoonFourMars;
+                ToastControl.Current.HideToastIn = 5;
+                ToastControl.Current.ToastAction = async () =>
+                {
+                    PageTitle = "Hello, Mercury!";
+                    await Task.Delay(1500);
+                    PageTitle = "Hello, World!";
+                };
+                ToastControl.Current.Text = parameter.ToString();
+
+                ToastControl.Current.DisplayToast();
```

Comment:
I don't think ToastControl looks good from a API consumer standpoint. Have you tried creating a extension method for `Application.Current.MainPage` to get `Application.Current.MainPage.DisplayToast`? I think that will feel more natural from a consumer

Reply:
`ToastControl` is an alias here, the call now would be like `DIPS.Xamarin.UI.Controls.Toast.Toast.Current.DisplayToast();` which is quite common when you try other 3rd party libraries

I didn't try to create an extension for `Application.Current.MainPage` at this stage

Reply:
Alias? Does that mean that you plan to change it?

Reply:
I am not familiar with the 3rd parties you mention and I don't think we should let other libraries drive our API. 

@Illedan what do you think in term of API?

The way we do it here will be the way we do it for all future static API? 
1. Keep it like @rizanzaky says with using the DXUI namespace? 
2. Extend MainPage to end up with a API that looks like Xamarin.Forms API?

Reply:
alias, this is the alias `using ToastControl = DIPS.Xamarin.UI.Controls.Toast.Toast;`

Reply:
Ah, sorry didn't see that. 👍

Reply:
After testing the api today I can say that it felt natural that it's a part of the controls namespace
It is a UI control that pops up in the screen, so it felt natural that it was under the namespace. Closing this conversation. 👍